contrib/clojure-package/test/org/apache/clojure_mxnet/image_test.clj

```diff
@@ -76,4 +77,14 @@
   (let [img-arr (image/read-image image-path)
         resized-arr (image/resize-image img-arr 224 224)
         new-img (image/to-image resized-arr)]
-    (is (= true (ImageIO/write new-img "png" (io/file tmp-dir "out.png"))))))
+    (is (true? (ImageIO/write new-img "png" (io/file tmp-dir "out.png"))))))
+
+(deftest test-draw-bounding-box!
+  (let [new-img  (-> (ImageIO/read (new File image-path))
+                     (image/draw-bounding-box! [{:x-min 190 :x-max 850 :y-min 50 :y-max 450}
+                                                {:x-min 200 :x-max 350 :y-min 440 :y-max 530}]
+                                               {:stroke 2
+                                                :names ["pug" "cookie"]
+                                                :transparency 0.8
+                                                :font-size-mult 2.0}))]
+    (is (true? (ImageIO/write new-img "png" (io/file tmp-dir "out.png"))))))
```

Comment:
`true?` is not needed here I think?

Reply:
We could also probably test that `new-img` and the `(ImageIO/read (new File image-path))` are not equal in terms of pixel values since a Bounding Box should be drawn?